src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.Dto.GroupDto;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collections;
+import java.util.List;
+
+@RequestMapping("v1/groups")
+@RestController
+@CrossOrigin("*")
 public class GroupController {
+
+    @GetMapping()
+    public ResponseEntity<List<GroupDto>>getGroups(){
```

Comment:
Nazwy DTO do podmiany na duże litery. To samo zrobię u siebie

Reply:
Napisz gdy już zmienisz

Reply:
Wszystko zmienione